lib/events/dynamoevents/dynamoevents.go

```diff
@@ -479,6 +609,19 @@ func (l *Log) GetSessionEvents(namespace string, sid session.ID, after int, inlc
 	return values, nil
 }
 
+// daysBetween returns a list of all dates between `start` and `end` in the format `yyyy-mm-dd`.
+func daysBetween(start time.Time, end time.Time) []string {
+	var days []string
+	oneDay := time.Hour * time.Duration(24)
+
+	for start.Before(end) {
+		days = append(days, start.Add(oneDay).Format(isoDateLayout))
+		start = start.Add(oneDay)
+	}
```

Comment:
this skips the date of `start`
if you call `daysBetween` with 2021/04/27-14:25 and 2021/04/29-13:00, you get `2021-04-28, 20201-04-29`; should this list include `2021-04-27` too?

Reply:
Updated loop to not take time of day into account which fixes this.